streamx-plugin/streamx-flink-packer/src/main/scala/com/streamxhub/streamx/flink/packer/maven/MavenRetriever.scala

```diff
@@ -52,7 +54,7 @@ object MavenRetriever {
   /**
    * default maven local repository
    */
-  lazy val localRepo = new LocalRepository(MAVEN_LOCAL_DIR)
+  lazy val localRepo = new LocalRepository(new Workspace(StorageType.LFS).MAVEN_LOCAL_DIR)
```

Comment:
I think it would be better to replace `new Workspace(StorageType.LFS)` with  `WorkspaceGetter.local` or  `WorkspaceGetter.get(StorageType.LFS) `,  `new XX` is more like getting a new instance of an object.

Reply:
I actually prefer the more concise `WorkspaceGetter.local`